include/dr/mhp/algorithms/inclusive_exclusive_scan_impl.hpp

```diff
@@ -57,8 +59,32 @@ auto inclusive_exclusive_scan_impl_(R &&r, O &&d_first, BinaryOp &&binary_op,
   using value_type = U;
   assert(aligned(r, d_first));
 
-  bool use_sycl = mhp::use_sycl();
   auto comm = default_comm();
+  bool use_sycl = mhp::use_sycl();
```

Comment:
why 'use_sycl' moved line down?
if there is no reason, then let's don't change code to have easier history